.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+- "3.5"
+  
+before_install:
+- pip install --no-index --trusted-host travis-wheels.scikit-image.org --find-links=http://travis-wheels.scikit-image.org numpy scipy
+  
+# command to install dependencies
+# install: "pip install -r requirements.txt"
+install: 
+# - pip install numpy
+# - pip install scikit
+- pip install librosa
```

Comment:
Add librosa to requirements.txt, and change this line to `pip install -r requirements.txt`.  Then, remove the comment above referring to this instruction.